chain-signatures/node/src/protocol/presignature.rs

```diff
@@ -530,6 +696,14 @@ impl PresignatureManager {
             }
         });
 
+        for presignature in new_presignatures {
+            self.insert(presignature).await;
+        }
+
+        for presignature in new_mine_presignatures {
+            self.insert_mine(presignature).await;
+        }
```

Comment:
not important for now, but would it better if we could insert all of them into redis at once instead of one by one?

Reply:
It is possible in Redis. I may add it later with atomicity PR (ability to read, write, and modify Redis state in a single atomic transaction).